test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/EnumTypeModelBinderProviderTest.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Binders;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
+{
+    public class EnumTypeModelBinderProviderTest
```

Comment:
Not a big deal but should probably add negative tests e.g. make sure no binder is returned for non-`enum` types. See `SimpleTypeModelBinderProviderTest.Create_ForCollectionOrComplexTypes_ReturnsNull(...)`

Reply:
Don't see where this has been addressed.